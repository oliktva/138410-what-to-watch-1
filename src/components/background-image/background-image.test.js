import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import BackgroundImage from './background-image';

Enzyme.configure({adapter: new Adapter()});

it(`renders correctly`, () => {
  const backgroundImage = shallow(<BackgroundImage image="test" name="test name" />);

  expect(toJson(backgroundImage)).toMatchSnapshot();
});
