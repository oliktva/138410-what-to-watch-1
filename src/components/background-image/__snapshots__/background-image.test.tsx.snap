// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="test name"
    src="test"
  />
</div>
`;
